benchopt/plotting/generate_html.py

```diff
@@ -163,6 +167,18 @@ def get_metadata(df, plot_configs):
     return metadata
 
 
+def get_custom_plots_for_html(df, benchmark):
+    benchmark.check_custom_plots()
+    custom_data = {}
+    custom_dropdown = {}
+    for plot in benchmark.get_custom_plots():
+        data, dropdown = plot._get_all_plots(df)
+        custom_data[plot._get_name()] = data
+        custom_dropdown[plot._get_name()] = dropdown
+
+    return custom_data, custom_dropdown
```

Comment:
Would move this to `Benchmark.get_all_data_plots`, so here we only handle rendering the data?